hazelcast/src/main/java/com/hazelcast/config/PermissionConfig.java

```diff
@@ -70,124 +100,126 @@ public enum PermissionType {
         /**
          * All
```

Comment:
Not related to your change, but not exactly helpful Javadoc there.

Reply:
I'll send an update commit for the `ALL` constant.
If you meant all JavaDocs within the enum, I would leave it to another PR.